Dockerfile

```diff
@@ -1,12 +1,25 @@
 FROM golang:1.13.8-alpine3.10 AS gobuild
 
+# CHAIN argument defines the chain implementation which should be used
+# during image build process.
+ARG CHAIN=ethereum
```

Comment:
Given that the client will soon talk to Bitcoin as well, I'd consider renaming `CHAIN` to `HOST_CHAIN` to be a little more specific. Especially that we mention the "host chain" later in this file in the comments.

Reply:
Please see https://github.com/keep-network/keep-ecdsa/commit/c1fec15c6b018f1049e970b6937a6f92387d1edc